### PR TITLE
Removed commandline openhab-generator closes: #329

### PR DESCRIPTION
**Pull request by EliasGabrielsson**
Signed-off-by: Elias Gabrielsson <elias@benefactory.se> (github: EliasGabrielsson)

### CODE REVIEW

File: docs/openhabian.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ You'll find all of these in the [openHABian Configuration Tool](#openhabian-conf
 
 - [Amanda Network Backup](http://www.amanda.org/) - A built-in backup solution for your valuables. Please continue reading [here](https://github.com/openhab/openhabian/blob/master/docs/openhabian-amanda.md)
 - [frontail](https://github.com/mthenw/frontail) - openHAB Log Viewer accessible from [http://openHABianPi:9001](http://openHABianPi:9001)
-- [openHAB Generator](https://github.com/kubawolanin/generator-openhab) - Allows you to create [Items](http://docs.openhab.org/configuration/items.html), [Sitemap](http://docs.openhab.org/configuration/sitemaps.html) and [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html) dashboard for your home in no time
+[Sitemap](http://docs.openhab.org/configuration/sitemaps.html) and [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html) dashboard for your home in no time
```
Comment (kubawolanin) [inline docs/openhabian.md]: This line should be removed as well

File: docs/openhabian.md [hunk 2]
```diff
@@ -331,7 +331,7 @@ If the installation was **not successful** you will see a warning and further in
 ##### What Next?
 
 After your first setup is done, you might want to generate [Items](http://docs.openhab.org/configuration/items.html), [Sitemap](http://docs.openhab.org/configuration/sitemaps.html) and [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html) dashboard for your home.
-Simply log in to your Raspberry PI with SSH and type `openhab-generator`. It will run a Yeoman-based [openHAB Generator](https://github.com/kubawolanin/generator-openhab) tool asking for your house structure (e.g. number of floors, rooms and devices).
+Simply checkout the generator which could be found at the openHAB dashboard.
```
Comment (kubawolanin) [inline docs/openhabian.md]: I would mention [Home Builder](https://docs.openhab.org/configuration/homebuilder.html) here as it's a natural evolution of `openhab-generator` in form of UI add-on.

**Comment by ThomDietrich** [comment]
Hey @EliasGabrielsson and @kubawolanin,
I've updated the Readme. could you do a quick check? Thanks!

**Comment by kubawolanin** [comment]
LGTM @ThomDietrich :)

**Comment by EliasGabrielsson** [comment]
+1 👍

**Pull request merged**